service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandler.java

```diff
@@ -323,14 +372,126 @@ private List<Element<SupportingEvidenceBundle>> getEvidenceBundle(CaseData caseD
         }
     }
 
-    private List<Element<SupportingEvidenceBundle>> getEvidenceBundleFromRespondentStatements(CaseData caseData) {
+    private Map<DocumentUploadNotificationUserType, List<FurtherDocument>>
+        getNotificationUserType2NewFurtherDocumentMap(CaseData caseData, CaseData beforeCaseData) {
+
+        // initialisation
+        Map<DocumentUploadNotificationUserType, List<FurtherDocument>> ret = new HashMap<>();
+        if (ret.get(ALL_LAS) == null) {
+            ret.put(ALL_LAS, new ArrayList<>());
+        }
+        if (ret.get(CAFCASS) == null) {
+            ret.put(CAFCASS, new ArrayList<>());
+        }
+        if (ret.get(CHILD_SOLICITOR) == null) {
+            ret.put(CHILD_SOLICITOR, new ArrayList<>());
+        }
+        if (ret.get(RESPONDENT_SOLICITOR) == null) {
+            ret.put(RESPONDENT_SOLICITOR, new ArrayList<>());
+        }
```

Comment:
won't `ret.get(KEY) == null` always be true in this case as we just instantiated the hashmap?